pkg/util/sys/kmod.go

```diff
@@ -17,11 +17,41 @@ package sys
 
 import (
 	"fmt"
+	"os/exec"
+	"strings"
 
-	"github.com/rook/rook/pkg/util/exec"
+	pkgexec "github.com/rook/rook/pkg/util/exec"
 )
 
-func LoadKernelModule(name string, options []string, executor exec.Executor) error {
+func getKernelVersion() (string, error) {
+	var output []byte
+	cmd := exec.Command("uname", "-r")
+	output, err := cmd.Output()
+	out := strings.TrimSpace(string(output))
+	if err != nil {
+		return out, err
+	}
+
+	return out, nil
+}
+
+func IsBuiltinKernelModule(name string, executor pkgexec.Executor) (bool, error) {
+	kv, err := getKernelVersion()
+	if err != nil {
+		return false, fmt.Errorf("failed to get kernel version: %+v", err)
+	}
+
+	kv = fmt.Sprintf("/lib/modules/%s/modules.builtin", kv)
+	out, err := executor.ExecuteCommandWithCombinedOutput(false, "check builtin kmod", "cat", kv)
+	if err != nil {
+		return false, fmt.Errorf("failed to cat /lib/modules: %+v", err)
```

Comment:
Last thing, instead of using `/lib/modules` here use the `kv` variable in the `fmt.Errorf()` call, from above so the user knows where the rook-agent is looking.

Reply:
Ah, OK, make sense. Thanks!